packages/xo-server/src/api/backup-ng.js

```diff
@@ -131,8 +131,11 @@ runJob.params = {
   schedule: {
     type: 'string',
   },
-  vm: {
-    type: 'string',
+  vms: {
+    type: 'array',
+    items: {
+      type: 'string',
+    },
```

Comment:
You are breaking the API.

It's not a major issue (especially since it's a fairly recent method) but it should be done with care.